drivers/hih6130/hih6130.c

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (C) 2015 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_hih6130
+ * @{
+ *
+ * @file
+ * @brief       Device driver implementation for Honeywell HumidIcon Digital
+ *              Humidity/Temperature Sensors: HIH-6130/6131 Series
+ *
+ * @author      Joakim Gebart <joakim.gebart@eistec.se>
+ *
+ * @}
+ */
+
+#include <stdint.h>
+
+#include "hih6130.h"
+#include "periph/i2c.h"
+#include "timex.h"
+#include "vtimer.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+/* Humidity is stored in the first 2 bytes of data */
+#define HIH6130_HUMIDITY_DATA_LENGTH (2)
+/* Humidity + temperature data is 4 bytes long */
+#define HIH6130_FULL_DATA_LENGTH (4)
+/* Bit mask for the status bits in the first byte transferred */
+#define HIH6130_STATUS_MASK      (0xc0)
+/* Bit mask for the humidity data */
+#define HIH6130_HUMIDITY_MASK    (0x3fff)
+/* Temperature data is left adjusted within the word */
+#define HIH6130_TEMPERATURE_SHIFT (2)
+
+enum {
+    HIH6130_STATUS_OK           = 0x00,
+    /**
+     * stale data: data that has already been fetched since the last measurement
+     * cycle, or data fetched before the first measurement has been completed.
+     */
+    HIH6130_STATUS_STALE_DATA   = 0x40,
+    HIH6130_STATUS_COMMAND_MODE = 0x80,
+    HIH6130_STATUS_DIAGNOSTIC   = 0xc0,
+};
+
+/** @brief Delay between requesting a measurement and data becoming ready */
+static const timex_t measurement_delay = {
+        .seconds = 0, .microseconds = 50 * MS_IN_USEC, };
```

Comment:
So you want a delay of 50000 us here? I just stumbled upon the `MS_IN_USEC` macro here.